app/views/directives/deploy-image.html

```diff
@@ -60,6 +60,10 @@
                 </button>
               </span>
             </div>
+            <div ng-if="input.selectedProject.metadata.uid && !import.namespace" class="help-block">To pull a secured container image that is not from the internal registry, you must
```

Comment:
@sg00dwin It's not clear to me why we're checking `!import.namespace` in the `ng-if`

Reply:
My original thought was to not display the help text with the link to create a pull secret, when the Image Name option is disabled. But thinking about it, we should probably just show it.